extensions/wikia/AdEngine/js/video/videoFrequencyStore.js

```diff
@@ -0,0 +1,103 @@
+/*global define*/
+define('ext.wikia.adEngine.video.videoFrequencyStore', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.adLogicPageViewCounter',
+	'ext.wikia.adEngine.utils.time',
+	'wikia.cache',
+	'wikia.log'
+], function (adContext, pageViewCounter, timeUtil, cache, log) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 5 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)